oidc_apis/models.py

```diff
@@ -186,3 +188,15 @@ class Meta:
 
     def __str__(self):
         return "{obj}[{lang}]".format(obj=self.master, lang=self.language_code)
+
+
+def get_all_required_scopes_by_api_scopes(scopes):
```

Comment:
I don't think this needs to be public. It is also very much related to ApiScope class, so maybe it should be a classmethod there?

Reply:
e.g. `ApiScope._get_required_scopes`